README.md

```
@@ -72,23 +72,87 @@ Administrator   administrator_2_0       Coronasafe@123
+
+```env
+# Map different frontend origins to different backend URLs
+REACT_CARE_URL_MAP='{"http://localhost:4000": "https://careapi.ohc.network", "http://care.localhost": "https://careapi.ohc.network", "http://develop.localhost": "https://develop-api.ohc.network", "http://dev.localhost": "http://localhost:9000"}'
+
+# Fallback for any unmatched origins
+REACT_CARE_API_URL=https://careapi.ohc.network
+```
+
+**Set up Nginx reverse proxy:**
+
+#### On macOS
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)